content/patterns/radio/examples/radio-rating.html

```diff
@@ -25,31 +25,33 @@
       </ul>
     </nav>
     <main>
-      <h1>Rating Radio Group Example</h1>
+      <h1>評価ラジオグループの例</h1>
+      <!-- 翻訳元リビジョン: https://github.com/w3c/aria-practices/tree/962f6e771dccd4e3bbf8b83da717fb8d75b861da -->
 
       <section>
-        <h2>About This Example</h2>
+        <h2>この例について</h2>
         <img alt="" class="example-page-example-icon" src="../../../images/pattern-radio.svg">
         <p>
-          Following is an example of a rating input that demonstrates the <a href="../radio-group-pattern.html">Radio Group Pattern.</a>
-          The rating is indicated by the number of stars selected by the user.
+          以下は、<a href="../radio-group-pattern.html">ラジオグループパターン</a>を示す評価入力の例です。
+          評価は、利用者が選択した星の数によって示されます。
         </p>
-        <p>Similar examples include:</p>
+        <p>類似の例は次のとおりです。</p>
         <ul>
-          <li><a href="../../slider/examples/slider-rating.html">Rating Slider Example</a>: Horizontal slider that demonstrates using <code>aria-valuetext</code> to make it easy for assistive technology users to understand the meaning of the current value chosen on a ten-point satisfaction scale.</li>
-          <li><a href="radio-activedescendant.html">Radio Group Example Using aria-activedescendant</a>: Radio button group that uses <code>aria-activedescendant</code> for managing keyboard focus.</li>
-          <li><a href="radio.html">Radio Group Example Using Roving tabindex</a>: A radio button group that uses roving <code>tabindex</code> for managing keyboard focus.</li>
+          <li><a href="../../slider/examples/slider-rating.html">評価スライダーの例</a>: <code>aria-valuetext</code> を使用して、支援技術の利用者が 10 段階の満足度スケールで選択された現在の値の意味を理解しやすくする方法を示す水平スライダー。</li>
```

Comment:
demonstoratesを受ける名詞が必要ということなんでしょうけれど、「方法」は意訳が過ぎるかなあと……
日本語的にはバッサリカットした方が自然……？

```suggestion
          <li><a href="../../slider/examples/slider-rating.html">評価スライダーの例</a>: <code>aria-valuetext</code> を使用して、支援技術の利用者が 10 段階の満足度スケールで選択された現在の値の意味を理解しやすくする水平スライダー。</li>
```